### DISCUSSION TITLE
ILI6122 support

### DISCUSSION THREAD

**Discussion by jancha** [moononournation/Arduino_GFX#375] [General]
Hello,

You have listed ILI6122 as a supported device, however, I could not find such display class in the source code.

Can you please add it or point towards other display class that can be used as a substitute?

Thanks,
Jancha

**Comment by moononournation** [comment]
It should be supported by RGB Panel.

**Reply by jancha** [reply]
Thanks, it is, just required very specific params.

Here is how to configure it https://github.com/moononournation/Arduino_GFX/discussions/367#discussioncomment-7491824
